letters/letters.py

```diff
@@ -0,0 +1,52 @@
+import re
+
+from redbot.core import commands
+
+
+# Define numbers -> emotes tuple
+nums = (':zero:', ':one:', ':two:', ':three:', ':four:', ':five:', ':six:', ':seven:', ':eight:', ':nine:')
+
+class Letters(commands.Cog):
+    """Letters cog"""
+
+    @commands.command()
+    async def letters(self, ctx, *, msg):
+        """Outputs large emote letters (\"regional indicators\") from input text.
+        Accepts a-z, 0-9, and whitespace only.
+
+        Example:
+        - `[p]letters I'd like this text as emotes 123`
```

Comment:
```suggestion
        - `[p]letters I would like this text as emotes 123`
```
Correct text to be valid command input

Reply:
LOL. Fixed in 665384e.